### ISSUE TITLE
How to actually get started?

### ISSUE THREAD

**Issue by Arachnosuchus** [danzilberdan/obsidian-mathlive#8]
I might be too incompetent to get this installed. 
How do I use the image scanning feature? Why does signing in not actually do anything? I purchased tokens. I put in the API key in Obsidian. Why does it fail to scan my clipboard? Is this some kind of scam? Well, congrats. Either way, you're five bucks richer, I guess.

**Comment by danzilberdan** [comment]
Hi definitely not a scam but maybe a bug on my end. Did you try to scan the image with snipping tool and then click the scan button?

I would like to make sure it works for you by doing a quick call and understanding the issue. Can you send me an email to dan.zilberman99@gmail.com? I will try to debug and fix the issue.

**Comment by Arachnosuchus** [comment]
Okay so, it seems to be working now? Is there supposed to be a waiting time?
Now my only question would be why it's so tiny. lol

![2024-06-18 22_43_50-Numerik-Aufgaben (Springer) - Obsidian Vault - Obsidian v1 6 3](https://github.com/danzilberdan/obsidian-mathlive/assets/78440622/512eb49e-aa9e-43cb-ac3f-65cc3e1ca2ed)

**Comment by Arachnosuchus** [comment]
Okay, I don't know why it wasn't working before. I sure didn't change anything. Maybe it needed some warming up before being able to scan.
Either way, it seems to be working now.

**Comment by danzilberdan** [comment]
This is probably because of the \scriptstyle tag in the beginning of the latex formula. Try to remove it and see if the equation is rendered in the correct size.
This is due to the fact that a machine learning model is running in the background outputting non-deterministic results. For me, this never happened but let me know if it repeats and maybe this is something that I can solve.

Regarding the time. For me, it usually takes about 2 seconds to resolve a scan.. How much time does it take for you?

**Comment by Arachnosuchus** [comment]
I think there was a rogue dot at the end of the equation which switched it to inline shape. When I removed it, it became normal.

Yes, the scanning takes a few seconds, which I expected. But before it wouldn't scan at all. It told me to look in the console and there it said it couldn't find a file or something. I just kept trying to scan maybe a few dozen times because I didn't know what else to do. After a few minutes it suddenly did a successful scan. I thought it better not to question it.

I might have also entered the wrong API key at least once. I tried several. In any way, it was likely due to human error on my part as I am not very familiar with Obsidian yet. Sorry for implying this is a scam. I was frustrated. c:
